internal/wasm/memory_test.go

```diff
@@ -828,6 +828,14 @@ func Test_atomicStoreLength(t *testing.T) {
 	require.Equal(t, 15, len(slice))
 }
 
+func Test_atomicStoreLengthAndCap(t *testing.T) {
+	// Doesn't verify atomicity, but at least we're updating the correct thing.
+	slice := make([]byte, 10, 20)
+	atomicStoreLengthAndCap(&slice, 15, 20)
```

Comment:
Please change the capacity as well. Something like `atomicStoreLengthAndCap(&slice, 12, 18)`.

Reply:
Oops, done